scripts/BluesHouse.asm

```diff
@@ -46,12 +46,12 @@ BluesHouseDaisySittingText:
 	SetEvent EVENT_GOT_TOWN_MAP
 	jr .done
 
-.got_town_map
+.gotTownMap
```

Comment:
I'm definitely fine with changing `.Done` to `.done` etc, but I've gotten real used to the easier readability of purely lowercase `.snake_case` labels. I'm not sure I'm ready to see it go.